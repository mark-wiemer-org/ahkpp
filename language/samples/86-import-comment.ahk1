; [Issue #86](https://github.com/mark-wiemer-org/ahkpp/issues/86)
#Include
#Include File.ahk
#Include File.ahk ;Comment
#Include C:\My Scripts
#Include C:\My Scripts ;Comment
