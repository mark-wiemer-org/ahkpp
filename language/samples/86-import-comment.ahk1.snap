>; [Issue #86](https://github.com/mark-wiemer-org/ahkpp/issues/86)
#^ source.ahk comment.line.semicolon.ahk punctuation.definition.comment.ahk
# ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^ source.ahk comment.line.semicolon.ahk
>#Include
#^^^^^^^^ source.ahk keyword.control.import.ahk
>#Include File.ahk
#^^^^^^^^ source.ahk keyword.control.import.ahk
#        ^^^^^^^^^ source.ahk keyword.control.import.ahk string.parameter.import.ahk
>#Include File.ahk ;Comment
#^^^^^^^^ source.ahk keyword.control.import.ahk
#        ^^^^^^^^^ source.ahk keyword.control.import.ahk string.parameter.import.ahk
#                 ^^^^^^^^^ source.ahk keyword.control.import.ahk comment.line.semicolon.import.ahk
>#Include C:\My Scripts
#^^^^^^^^ source.ahk keyword.control.import.ahk
#        ^^^^^^^^^^^^^^ source.ahk keyword.control.import.ahk string.parameter.import.ahk
>#Include C:\My Scripts ;Comment
#^^^^^^^^ source.ahk keyword.control.import.ahk
#        ^^^^^^^^^^^^^^ source.ahk keyword.control.import.ahk string.parameter.import.ahk
#                      ^^^^^^^^^ source.ahk keyword.control.import.ahk comment.line.semicolon.import.ahk
>